canonical-websites/jp.ubuntu.com
# 50 kubernetes

## Done
Build the kubernetes page

## QA
- Go to /kubernetes
- Check the content matches [the copy doc](https://docs.google.com/document/d/1t-fQU3HfRNRVoIRjsFJRyDbHq-NvP7YapdnijAqBtNI/edit#heading=h.yjalla1b1o96)

## Issue / Card
Fixes https://github.com/canonical-websites/jp.ubuntu.com/issues/50
